src/fmu/dataio/export/rms/inplace_volumes.py

```diff
@@ -26,28 +26,31 @@
 _logger: Final = null_logger(__name__)
 
 
+VolumetricColumns = _enums.InplaceVolumes.VolumetricColumns
+TableIndexColumns = _enums.InplaceVolumes.TableIndexColumns
```

Comment:
Should probably private these unless we explicitly want to users to use them

Reply:
thanks, I always forget 👍